### PR TITLE
Development Automation with Docker

### PR DESCRIPTION
**Pull request by hungnguyen3**
### Added `docker-compose` file to spin up:
- client
- scraper
- postgresql

### CODE REVIEW

File: scraper/docker-compose.yml
Comment (sbshaun) [inline scraper/docker-compose.yml:1]: why delete this?

Comment (hungnguyen3) [reply scraper/docker-compose.yml:1]: I moved it outside of the folder to the `./`:

look here: [link](https://github.com/jobless-devs/Jobhub/pull/5/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3)

Comment (sbshaun) [reply scraper/docker-compose.yml:1]: I'm not sure if it's needed. To my understanding, 2 containers can communicate, and that's why they should be defined in a docker-compose. The scraper only needs to run for a while, so I feel it's a bit of unnecessary?

Comment (hungnguyen3) [reply scraper/docker-compose.yml:1]: 2 points:
- The scraper should be made so it runs periodically later -> not exactly `only run for a while` -> better to keep the container alive
- 1 docker-compose to spin up all the services: client, scraper, backend, database, etc. -> arguably better than having multiple files in all folders?

Comment (sbshaun) [reply scraper/docker-compose.yml:1]: 2 questions: 
1. Is this just to make development easier, I was thinking we'll deploy each part separately in the end.  
2. there are 2 columns but only 1 is used? `jobhub-client-sync`

Comment (hungnguyen3) [reply scraper/docker-compose.yml:1]: for `development`, we use `docker-compose`

for `production`, we build docker images -> push to docker hub  -> use `k8s` to deploy these `images`

Comment (hungnguyen3) [reply scraper/docker-compose.yml:1]: > 2 questions:
> 
> 1. Is this just to make development easier, I was thinking we'll deploy each part separately in the end.
> 2. there are 2 columns but only 1 is used? `jobhub-client-sync`

Nice catch! LOL I copied and pasted should be `jobhub-scaper-sync`

**Review by sbshaun — APPROVED** [review]
lgtm!

**Pull request merged**